docs/source/man.rst

```diff
@@ -126,36 +126,36 @@ Credentials
 
 Use the ``qpc cred`` command to create and manage credentials.
 
-A credential contains user name and password or SSH key information that is used for authentication during a scan. The Quipucords tool uses SSH to connect to servers on the network and uses credentials to access those servers.
+A credential contains user name and password or SSH key information that is used for authentication during a scan. Certain credential types also support the use of an access token as an alternative authentication method. The Quipucords tool uses SSH to connect to servers on the network and uses credentials to access those servers.
 
 When a scan runs, it uses a source that contains information such as the host names, IP addresses, a network, or a systems management solution to be accessed. The source also contains references to the credentials that are required to access those systems. A single source can contain a reference to multiple credentials as needed to connect to all systems in that network or systems management solution.
 
 Creating and Editing Credentials
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create a credential, supply the type of credential and supply SSH credentials as either a user name-password pair or a user name-key pair. The Quipucords tool stores each set of credentials in a separate credential entry.
+To create a credential, supply the type of credential and supply SSH credentials as either a user name-password pair or a user name-key pair. Additionally, for Openshift credentials, you can also use an access token as an alternative. The Quipucords tool stores each set of credentials in a separate credential entry.
```

Comment:
The new phrase seems out of place to me. The original text is referring to credentials in a generic way - it's not pointing specifics for vcenter and satellite (which do not use ssh credentials for that matter). I think pointing that access token is one of the options is enough - OR we explain in detail specifics for each source type.

Reply:
I moved the new sentence and added token as one other option without specifics about cred types.